File: eva-core/src/main/java/io/nerv/properties/Security.java
package io.nerv.properties;

import lombok.Data;

/**
 * 访问鉴权配置
 */
@Data
public class Security {
    // 无需鉴权路径(无条件访问)
    private String[] anonymous;

    // 无需资源鉴权的路径
    private String[] permit;
}

File: eva-core/src/main/kotlin/io/nerv/properties/Security.kt
package io.nerv.properties

/**
 * 访问鉴权配置
 */
class Security {
    // 无需鉴权路径(无条件访问)
    var anonymous: Array<String> = Array(1) {""}

    // 无需资源鉴权的路径
    var permit: Array<String> = Array(1) {""}
}